webrtc-stats.html

```diff
@@ -3016,9 +3016,13 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  The URL of the TURN or STUN server indicated in the that translated this IP
-                  address. It is the URL address surfaced in an
-                  <code>RTCPeerConnectionIceEvent</code>.
+                  For local candidates this is the URL of the ICE server from
+                  which the candidate was obtained. It is the same as the
+                  <code>url</code> surfaced in the
```

Comment:
Please link to http://w3c.github.io/webrtc-pc/#dom-rtcpeerconnectioniceevent-url otherwise we get a markup loop.